### DISCUSSION TITLE
fzf border vs fzf-lua border

### DISCUSSION THREAD

**Discussion by Masber** [ibhagwan/fzf-lua#1481] [Q&A] [answered]
I realized one could either ask fzf to print the border or delegate this to fzf-lua, are these 2 options doing the same or are implementations different? if they are different, which one should we use from performance perspective?

**Comment by ibhagwan** [comment, marked as answer]
> I realized one could either ask fzf to print the border or delegate this to fzf-lua, are these 2 options doing the same or are implementations different? if they are different, which one should we use from performance perspective?

These aren't the same, fzf border will be drawn by fzf in the terminal whereas the fzf-lua border is drawn by the neovim floating window itself (the default), I don't think there's a difference in performance but I would use the default (drawn by fzf-lua/neovim), the main 2 reasons are:
(1) Ability to set the highlight `FzfLuaBorder`
(2) Ability to use the title instead of the prompt (try `:FzfLua profiles` and try `default-title` or `borderless-full` to see the difference, the title will be used instead of the prompt to indicate which picker is used
